src/main/java/com/snowflake/kafka/connector/internal/streaming/SchematizationUtils.java

```diff
@@ -220,7 +243,11 @@ private static String convertToSnowflakeType(Type kafkaType) {
       case STRING:
         return "VARCHAR";
       case BYTES:
-        return "BINARY";
+        if (Decimal.LOGICAL_NAME.equals(schemaName)) {
+          return "VARCHAR";
+        } else {
+          return "BINARY";
+        }
```

Comment:
it would be good to mention in description that it is not decimal to varchar but the bytes that represents decimal. Is that correct understanding?